### ISSUE TITLE
Integrate lv600s

### ISSUE THREAD

**Issue by CLARENNE-Q** [formatBCE/HA_VeSync_Classic300s#10]
Hello,

I'am using thing custom integration, it's work really fine ! Thanks a lot for your work.

I have also lv600s and it's look supported by pyvesync  https://github.com/webdjoe/pyvesync/pull/112

Can you please integrate on your side ?

Quentin

**Comment by formatBCE** [comment]
Yeah, should work. I doubt I will find time to integrate it in the nearest future. If you don't want to do it and create pull-request, I will pick it up in few weeks.

**Comment by CLARENNE-Q** [comment]
<img width="1539" alt="Capture d’écran, le 2023-01-09 à 22 12 53" src="https://user-images.githubusercontent.com/1670151/211453757-95e80f5b-a1cf-484d-b85e-86015f01650f.png">

Only 300S is working for the moment :(

**Comment by formatBCE** [comment]
I mean, library should work, so it's possible to integrate. :)

**Comment by CLARENNE-Q** [comment]
Hello FormatBCE,

I tried to add reference in all files for the lv600s and restart ha:

<img width="910" alt="Capture d’écran, le 2023-01-10 à 20 52 21" src="https://user-images.githubusercontent.com/1670151/211699576-bb4f2950-932e-4583-93a8-ecb98e8692f1.png">

Not detected, must I need do something else ?

Quentin

**Comment by formatBCE** [comment]
Nice job!
1. AFAIK this device doesn't support night light (at least pyvesync doesn't). So it won't work and can give errors.
2. I believe, pyvesync version should be bumped. Check please latest pyvesync release and update dependency.

After that, I guess, it should work.

**Comment by formatBCE** [comment]
@CLARENNE-Q did you manage to do it? I accept PRs :)
